gamemode/sv_chatmsg.lua

```diff
@@ -0,0 +1,23 @@
+-- The file provides the functionality for sending chat messages to players from
+-- the server. This is used instead of PrintMessage because we want to have
+-- colored messages (PrintMessage can't do this).
+
+
+util.AddNetworkString("ph_chatmsg")
+local PlayerMeta = FindMetaTable("Player")
+
+
+-- Sends a message to an individual player.
+function PlayerMeta:PlayerChatMsg(...)
```

Comment:
This stutters, just make it `ChatMsg`.

Reply:
I think it becomes more handy when you're actually calling this. Most of the calls are of the form `self:PlayerChatMsg()`, but if you still think `self:ChatMsg()` is a better form I can change it?